src/UI/Implementation/Component/Input/Field/Group.php

```diff
@@ -154,4 +154,10 @@ protected function getDataFactory(): DataFactory
     {
         return $this->data_factory;
     }
+
+    /** ATTENTION: see trait Group::_isClientSideValueOk! */
```

Comment:
Could we make this an annotation?
```suggestion
    /** ATTENTION: @see GroupInternals::_isClientSideValueOk() */
```